test/Microsoft.AspNet.Mvc.FunctionalTests/WebApiCompatShimBasicTest.cs

```diff
@@ -275,32 +275,6 @@ public class WebApiCompatShimBasicTest
             Assert.Equal(38, response.Content.Headers.ContentLength);
         }
 
-        [Fact]
-        public async Task ApiController_ResponseReturned_Chunked()
```

Comment:
I removed this test on purpose as when this test is run on real servers, it could fail based on their own respective behaviors...if we come up with the expectations on how all hosts work, then we could enable this test and add more tests...thoughts?